gradle.properties

```diff
@@ -1,5 +1,5 @@
 VERSION_CODE=1
```

Comment:
I've followed Felipe's example in other open source libraries and don't see this done. I'm not actually sure what this version code does but it doesn't seem used. I"ll talk to felipe